lua/windows.lua

```diff
@@ -7,23 +7,26 @@
 --    require("windows")
 --  end
 
--- We want to use Powershell 7.x (pwsh.exe)
--- if it is available!
-vim.cmd(":silent !pwsh.exe --version")
-if vim.v.shell_error == 0 then
-  vim.opt.shell = 'pwsh.exe'
+if vim.loop.os_uname().system == "Windows" then
```

Comment:
I guess the OS check is already done in the init.lua. Do you need it also in the windows.lua?

Reply:
Yeah right. I was just in a hurry to make it work and I overlooked the check in init.lua. Strangely the check doesnt even work for me because `vim.fn.has("win32")` is true for me on Linux(?).
`vim.loop.os_uname().system == "Windows"` also doesnt actually work ... it returns nil. 
Great.